source/_integrations/discovery.markdown

```diff
@@ -98,17 +98,17 @@ Valid values for enable are:
 
 ## Troubleshooting
 
-### UPnP
+### mDNS and UPnP
 
 Home Assistant must be on the same network as the devices for UPnP discovery to work.
-If running Home Assistant in a [Docker container](/docs/installation/docker/) use switch `--net=host` to put it on the host's network.
+When running Home Assistant in a [Docker container](/docs/installation/docker/) comand line option `--net=host` or the compose file equivalent `network_mode: host` must be used to put it on the host's network, otherwise mDNS and UPnP will not work.
```

Comment:
```suggestion
When running Home Assistant Core in a [Docker container](/docs/installation/docker/) command line option `--net=host` or the compose file equivalent `network_mode: host` must be used to put it on the host's network, otherwise mDNS and UPnP will not work.
```